content/en/docs/instrumentation/erlang/manual.md

```diff
@@ -2,12 +2,24 @@
 title: Manual
 aliases: [/docs/instrumentation/erlang/instrumentation]
 weight: 30
+description: Manual instrumentation for OpenTelemetry Erlang/Elixir
 ---
 
-Instrumentation is the act of adding observability code to your application.
-This can be done with direct calls to the OpenTelemetry API within your code or
-including a dependency which calls the API and hooks into your project, like a
-middleware for an HTTP server.
+[Manual instrumentation](https://opentelemetry.io/docs/concepts/instrumentation/manual/)
+is the process of adding observability code to your application yourself. If you
+are developing a standalone process or service, then you would take a dependency
```

Comment:
I know its a different meaning but the use of "process" here is a little weird in the context of Erlang. Can we do just `standalone service` or `standalone program`?

Reply:
Right. I'd vote to just drop "process".

Reply:
would application work?

Reply:
Application also has a name that's used differently in Erlang. We have "applications" and "library applications" as the two types of 'collections of modules' sets of dependencies in the OTP framework (the former being stateful and the latter being stateless), and therefore a single "service" or end-user application can be made of multiple "OTP applications." 

These standards are confusing, but within the ecosystem they've been in use since the mid 1990s.

"standalone program" or "standalone service" are better words that do not clash within the Erlang/OTP ecosystem and that every user there will understand properly.

Reply:
it's service now :-)